### ISSUE TITLE
Change Fonts and give it proper styles

### ISSUE THREAD

**Issue by minhaj-313** [minhaj-313/Portfolio-1#42]

**Comment by rushikesh1799** [comment]
Hi Minhaj, I am new to open source and I would like to contribute to your portfolio-1. Can you please assign this to me and specify more on which section fonts you would like to change? 

Thanks.

**Comment by minhaj-313** [comment]
> Hi Minhaj, I am new to open source and I would like to contribute to your portfolio-1. Can you please assign this to me and specify more on which section fonts you would like to change?
> 
> Thanks.

Yeah you can do it change the font of all portfolio but give separate font means for heading give different and for paragraph give some different. make sure this fonts look attractive then only I will merge your PR as hactoberfest-accepted

**Comment by minhaj-313** [comment]
> Hi Minhaj, I am new to open source and I would like to contribute to your portfolio-1. Can you please assign this to me and specify more on which section fonts you would like to change?
> 
> Thanks.

first, you need to give a star to this repository and follow me then start doing as per hacktoberfest rule for more information you can read contributer.md file Thank You

**Comment by rushikesh1799** [comment]
Yes done, Can you please assign this to me so that I can start working on this one?

Thanks

**Comment by minhaj-313** [comment]
no remove it 
dont repeat it just do at once
